src/features/flows/nRF9151/Evaluate.tsx

```diff
@@ -34,7 +34,7 @@ import {
 } from './nrf9151Slice';
 
 const nRFCloudLink =
```

Comment:
as link changes, I'd rename it as well
```suggestion
const provisioningLink =
```